config/application.yml.default

```diff
@@ -291,6 +291,7 @@ session_total_duration_timeout_in_minutes: 720
 set_remember_device_session_expiration: false
 sp_handoff_bounce_max_seconds: 2
 show_user_attribute_deprecation_warnings: false
+min_attempts_remaining_warning_count: 3
```

Comment:
Since we have a lot of different types of rate limiting, I think it'd be good to have some sort of prefix which indicates that this is specific to OTP.

Maybe `otp_` ?

```suggestion
otp_min_attempts_remaining_warning_count: 3
```

There's also another related config prefixed with `login_otp_confirmation_`, but that could become quite long I think.

Reply:
Based on the name, I would expect the warning to start showing when there are 3 or fewer attempts remaining, but with the current implementation, it'll show at 2 attempts remaining. Could we update the implementation to match (use `>` instead of `>=`), or alternatively the name of the config to clarify that it's shown when there are fewer than this number?